internal/datanode/timetick_sender.go

```diff
@@ -144,10 +130,10 @@ func (m *timeTickSender) mergeDatanodeTtMsg() ([]*msgpb.DataNodeTtMsg, map[strin
 				commonpbutil.WithSourceID(m.nodeID),
 			),
 			ChannelName:   channelName,
-			Timestamp:     lastTs,
+			Timestamp:     segStats.ts,
 			SegmentsStats: toSendSegmentStats,
 		})
-		sendedLastTss[channelName] = lastTs
+		sendedLastTss[channelName] = segStats.ts
```

Comment:
this should be segment level rather than channel level?

Reply:
Since all segments with the same vchannel have the same time tick, the channel level is ok.